0_web_functions.R

```diff
@@ -85,7 +85,7 @@ set_web_parameters <- function(file_path) {
   twodii_internal <<- cfg$parameters$twodii_internal
   new_data <<- cfg$parameters$new_data
 
-  financial_timestamp <<- cfg$parameters$financial_timestamp
+  # financial_timestamp <<- cfg$parameters$financial_timestamp
```

Comment:
I have a general question,
I often see parts of code commented out instead of deleted and do not entirely understand the reasoning behind it. For me commented out code always signals that this is something that will become important again very soon and that one still somehow needs to have in the back of one`s mind. Is that the case here? :)

Reply:
see my original explanation above: "These code changes should probably be commented, and the commented code deleted, but for now I left it like this or brevity."

Reply:
i.e. I will not merge it like that... I just thought it was easier for people to review and give feedback on the change if it's super clear what has changed (depending on how they view it)

Reply:
ok, thx for the information